daml-lf/transaction-test-lib/src/main/scala/lf/value/test/TypedValueGenerators.scala

```diff
@@ -416,7 +417,10 @@ object TypedValueGenerators {
         : Arbitrary[HRec[Cid]]
     private[TypedValueGenerators] implicit def recshrink[Cid: Shrink]: Shrink[HRec[Cid]]
 
-    private[TypedValueGenerators] def injVar[Cid: IntroCtx](v: HVar[Cid]): (Ref.Name, Value[Cid])
+    private[TypedValueGenerators] def injVar[Cid: IntroCtx](
+        v: HVar[Cid],
+        rank: Int = 0,
```

Comment:
is the default here useful? this seems like it’s just asking for incorrect ranks.

Reply:
Indeed it's asking for things to go wrong.